From: Dmitry Smirnov <onlyjob@debian.org>
Date: Sun, 15 Nov 2020 21:42:17 +0800
Subject: skip privileged test: factory_linux_test.go

Last-Update: 2025-02-18
Forwarded: not-needed
---
 libcontainer/factory_linux_test.go | 2 ++
 1 file changed, 2 insertions(+)

Index: runc-1.3.3/libcontainer/factory_linux_test.go
===================================================================
--- runc-1.3.3.orig/libcontainer/factory_linux_test.go
+++ runc-1.3.3/libcontainer/factory_linux_test.go
@@ -25,6 +25,7 @@ func TestFactoryLoadNotExists(t *testing
 }
 
 func TestFactoryLoadContainer(t *testing.T) {
+	t.Skip("DM - skipping privileged test")
 	root := t.TempDir()
 	// setup default container config and state for mocking
 	var (
